### DISCUSSION TITLE
Z Axis Alarm Every Start

### DISCUSSION THREAD

**Discussion by HJTex** [winder/Universal-G-Code-Sender#2608] [Help]
Home machine no issue, initiate probe no issue, set zero no issue, start job and spindle immediately raises triggering alarm.  This is all after installing an extension kit, but no other issues or errors.  Any ideas on what Im missing?

**Comment by breiler** [comment]
You mean that it is moving the spindle so far up that it is triagering the limit switch?

Does your Gcode contain a G28 in the beginning? If so, you need to set the G28.1 on safe position. Every time the G28 is sent it will go to this position.

**Reply by HJTex** [reply]
Hello and thank you breiler.  You are correct, every time I start the job, the spindle moves up and tiggers the limit switch.  I;ll review the GCode and hopefully understand how to fix this.  I am very new to CNC machining, small 3018 that was working just fine until I installed the extension kit.

**Reply by breiler** [reply]
If it contains a G28, just run the command G28.1 after you've homed your machine and you should be ready to go.
